s3deploy.js

```diff
@@ -33,6 +33,10 @@ async function setupAWS(awsProfile, region) {
     }
   }
 
+  if (!endpoint) {
```

Comment:
```suggestion
  if (endpoint) {
```

Reply:
I think that conditional needs to be reversed.

Reply:
Thank you. That was indeed a mistake.